### PR TITLE
Maven Updates

### PR DESCRIPTION
**Pull request by agrosner**
1. Updated to provide maven support and convert project in correct gradle structure.
2. Depends on version 1.0.0 of BaseUtils
3. Added version 1.0.0 in [maven-releases](https://github.com/Raizlabs/maven-releases)

### CODE REVIEW

**Comment by dylanrjames** [comment]
Same concerns as in Base Utils. Nothing extra, but everything there carries over to here.

**Comment by agrosner** [comment]
@dylanrjames mostly package name updates, the gradle properties from `CoreUtils` pretty much carries over to here. updated version to 2.0.0

File: WebServiceManager/pom.xml [hunk 1]
```diff
@@ -2,16 +2,16 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <groupId>com.raizlabs.net.webservicemanager</groupId>
+    <groupId>com.raizlabs.android.net.webservicemanager</groupId>
     <artifactId>RZWebServiceManager</artifactId>
     <packaging>apklib</packaging>
-    <name>RZWebServiceManager</name>
+    <name>RZWebServiceManager</name>
```
Comment (dylanrjames) [inline WebServiceManager/pom.xml]: #nth Are we going to go through with a rename at this point? If we're re-packaging, I would say we should do this simultaneously.

Comment (agrosner) [inline WebServiceManager/pom.xml]: uh this may be legacy. The poms should not be included in the repo. This is here by mistake.

Comment (dylanrjames) [inline WebServiceManager/pom.xml]: Good point. We shouldn't need maven anymore anyway, so lets rip these out.

**Pull request closed without merging**